presto-common/pom.xml

```diff
@@ -37,6 +37,12 @@
             <artifactId>jol-core</artifactId>
         </dependency>
 
+        <dependency>
```

Comment:
This library is a part of our SPI, and we are trying to keep it clean of any dependencies. Is there an implementation in Java? If not - then we should probably implement it / copy it to the source.

Reply:
@arhimondr I now vendored in the code from apache, lmk if that works

Reply:
Sounds good